adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -159,10 +159,12 @@
                 var callback = parseCallback(window.location.href);
 
                 if (callback) {
-                    setupCheckLoginIframe();
-                    window.history.replaceState({}, null, callback.newUrl);
-                    processCallback(callback, initPromise);
-                    return;
+                    return setupCheckLoginIframe().success(function() {
+                        window.history.replaceState({}, null, callback.newUrl);
+                        processCallback(callback, initPromise);
+                    }).error(function (e) {
+                        throw 'Could not initialize iframe';
```

Comment:
I am not entirely sure what the right thing to do here is.  Do we throw or do we just console.error?  Looking for guidance.

Reply:
It should call initPromise.error() that way the code running init() will be able to handle the failure to load the iframe